app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,82 @@
+class Users::RegistrationsController < Devise::RegistrationsController
```

Comment:
When signing up I intentionally tried to put password different to confirm password. It stopped me from signing up but didnt say way, might be good to tell users why the sign up is failing. I noticed it was also failing for having a username and email set to "1". Is there a check for uniqueness and min lengths here or format?